themes/default/content/docs/get-started/aws/create-project.md

```diff
@@ -112,19 +112,27 @@ Saved config
 > What are [projects](/docs/intro/concepts/project/) and [stacks](/docs/intro/concepts/stack/)? Pulumi projects and stacks let you organize Pulumi code. Consider a Pulumi _project_ to be analogous to a GitHub repo---a single place for code---and a _stack_ to be an instance of that code with a separate configuration. For instance, _Project Foo_ may have multiple stacks for different development environments (Dev, Test, or Prod), or perhaps for different cloud configurations (geographic region for example). See [Organizing Projects and Stacks](/docs/guides/organizing-projects-stacks/) for some best practices on organizing your Pulumi projects and stacks.
 
 {{% choosable language "javascript,typescript" %}}
+
 After some dependency installations from `npm`, the project and stack will be ready.
```

Comment:
Added this whitespace because the Markdown `npm` isn't rendering properly without it:

![image](https://user-images.githubusercontent.com/274700/232936101-152d24c0-46e0-4768-a33a-35dc541d5dba.png)